UpdateDependencies.ps1

```diff
@@ -22,6 +22,16 @@ param(
 $LatestVersion = Invoke-WebRequest $VersionFileUrl -UseBasicParsing
 $LatestVersion = $LatestVersion.ToString().Trim()
 
+if ($DirPropsVersionElements -contains 'CoreClrExpectedPrerelease')
+{
+    # Also get list of all package versions, relative to the given prerelease version url.
+    $LatestPackagesListUrl = $VersionFileUrl -Replace 'Latest.txt', 'Latest_Packages.txt'
+    $LatestPackagesList = Invoke-WebRequest $LatestPackagesListUrl -UseBasicParsing
+    $LatestCoreCLRPackage = $LatestPackagesList -split "`n" | ?{ $_.StartsWith('Microsoft.NETCore.Runtime.CoreCLR') }
+    $LatestCoreCLRVersion = ($LatestCoreCLRPackage -split ' ')[1].Trim()
+}
```

Comment:
My vision is that `Latest.txt` definitely goes away as soon as we can.  And that `Latest_Packages.txt` turns into `Latest_BuildInfo.json` and we start writing more info into these files.